src/renderers/dom/server/ReactServerRenderingTransaction.js

```diff
@@ -7,12 +7,14 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactServerRenderingTransaction
+ * @flow
```

Comment:
I don’t have enough experience with this so I’m just passing on what @zpao said. To avoid bikeshedding this PR, let’s just remove `@flow` from any existing files in this PR, and get the PR itself through. You can later make an additional PR focused on Flow where we can discuss this and clarify with @zpao and others.